src/glang/lib/std/string.c3

```diff
@@ -32,7 +32,7 @@ function length : (sv : StringView&) -> isize = {
 }
 
 // TODO: return `UTF32Codepoint` here
```

Comment:
Intentionally ignored since I don't want to break the parser and conflict with myself rn.